src/utils/eventBus/index.ts

```diff
@@ -0,0 +1,28 @@
+class EventBus {
+  listeners: {} = {};
```

Comment:
Неплохо было бы написать типы для листенеров
```suggestion
  listeners: Record<string, (...args: unknown[]) => void>;
```